### PR TITLE
docs: Japanese translation 20201015

### PR DESCRIPTION
**Pull request by wahapo**
## Context
Update Japanese translation about [this comparison](https://github.com/screwdriver-cd/guide/compare/v0.0.301...v0.0.309).

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: docs/ja/cluster-management/configure-api.md [hunk 1]
```diff
@@ -554,6 +555,25 @@ webhooks:
...
 
+### レートリミット
+
+以下の環境変数を設定することで、認証トークンによるレートリミットを設定します。
+
+| 環境変数名           | デフォルト値  | 説明                 |
+|:---------------------|:--------------|:---------------------|
+| RATE_LIMIT_VARIABLES | `'{ "enabled": false, "limit": 300, "duration": 300000 }'` | レートリミットを設定するJSON文字列 |
+
+デフォルト値として、以下のような `config/local.yaml` で設定した値が使用されます。
```
Comment (yoshwata) [inline docs/ja/cluster-management/configure-api.md]: デフォルト値を上書きしたいときはlocal.yamlをつかってくださいみたいなほうが言いたいことなのかなと思いました。

File: docs/ja/user-guide/configuration/annotations.md [hunk 2]
```diff
@@ -58,6 +58,7 @@ jobs:
 | screwdriver.cd/dockerCpu | `MICRO` / `LOW` / `HIGH` / `TURBO` | k8s executorを利用していてDockerが有効な場合、Dockerコンテナで使用するCPU数を設定することができます。設定される値については`screwdriver.cd/cpu`の説明をご覧ください。 |
 | screwdriver.cd/dockerRam | `MICRO` / `LOW` / `HIGH` / `TURBO` | k8s executorを利用していてDockerが有効な場合、Dockerコンテナで使用するメモリ容量を設定することができます。設定される値については`screwdriver.cd/ram`の説明をご覧ください。 |
 | screwdriver.cd/coverageScope | `pipeline` / `job` | カバレッジプラグインを利用している場合に、プロジェクトを作成するスコープを設定できます。デフォルト値はクラスタの設定(e.g. `COVERAGE_SONAR_ENTERPRISE`)によるので、クラスタ管理者にご確認ください。 |
+| screwdriver.cd/displayName | パイプライングラフに表示するジョブ名 | パイプライングラフに表示するジョブ名を、yamlの制約に縛られない自由な名前にできます。 |
```
Comment (yoshwata) [inline docs/ja/user-guide/configuration/annotations.md]:
```suggestion
| screwdriver.cd/displayName | パイプライングラフに表示するジョブ名 | パイプライングラフに表示するジョブ名を、yamlのユニーク制約に縛られない自由な名前にできます。 |
```

**Review by yoshwata — APPROVED** [review]

File: docs/user-guide/local.md [hunk 3]
```diff
@@ -48,7 +48,6 @@ If you get the following error while running the update command,
 Error occurred while detecting version: GET https://api.github.com/repos/screwdriver-cd/sd-local/releases: 403 API rate limit exceeded.
 ```
 Please set the [GitHub personal access token.](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token)
-### bash
```
Comment (kumada626) [inline docs/user-guide/local.md:51]: Is it OK to delete this line?

Comment (wahapo) [reply docs/user-guide/local.md:51]: Maybe we can remove this line. I don't understand why it's there.

Comment (kumada626) [reply docs/user-guide/local.md:51]: 🙆‍♀️

**Review by kumada626 — APPROVED** [review]

**Pull request merged**